src/server/pfs/cmds/cmds.go

```diff
@@ -455,46 +454,49 @@ $ {{alias}} foo@XXX bar@YYY
 # return commits caused by foo@XXX leading to repos bar and baz
 $ {{alias}} foo@XXX -r bar -r baz`,
 		Run: cmdutil.Run(func(args []string) (retErr error) {
-			commits, err := cmdutil.ParseCommits(args)
-			if err != nil {
-				return err
-			}
-
-			c, err := client.NewOnUserMachine("user")
-			if err != nil {
-				return err
-			}
-			defer c.Close()
-
-			var toRepos []*pfsclient.Repo
-			for _, repoName := range repos {
-				toRepos = append(toRepos, client.NewRepo(repoName))
-			}
+			// TODO(global ids): implement flush job command
+			return errors.New("unimplemented")
```

Comment:
Just adding to a todo there because I don't think we have tests for most of pachctl